src/test/java/dasniko/testcontainers/keycloak/ExtendableKeycloakContainerTest.java

```diff
@@ -0,0 +1,63 @@
+package dasniko.testcontainers.keycloak;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.utility.MountableFile;
+
+class ExtendableKeycloakContainerTest {
```

Comment:
I think you can use `ParameterizedTest`s in this class, as all tests are the same, just with different input values!?

Reply:
Absolutely, thanks for the tip!

Changed the implementation in [this commit].

[this commit]: https://github.com/dasniko/testcontainers-keycloak/pull/112/commits/6b87cda46c4ba1822bf1e39d7761737316ce8616